### ISSUE TITLE
mozilla-esr38 jobs are busted because mozharness cannot be downloaded

### ISSUE THREAD

**Issue by whimboo** [mozilla/mozmill-ci#770]
Also mozilla-esr38 is affected here across all platforms:
https://treeherder.mozilla.org/#/jobs?repo=mozilla-esr38&filter-searchStr=Firefox%20UI&filter-tier=2&filter-tier=3&selectedJob=66555

```
2016-03-15 17:08:57,019 "default" was passed as the revision. Querying remote repository for corresponding rev hash of current default tip
Traceback (most recent call last):
  File "archiver_client.py", line 313, in <module>
    main()
  File "archiver_client.py", line 296, in main
    options, args = options_args()
  File "archiver_client.py", line 262, in options_args
    options.rev = custom_mozharness_options(options)
  File "archiver_client.py", line 286, in custom_mozharness_options
    new_rev = subprocess.Popen(cmd, stdout=subprocess.PIPE).communicate()[0].strip()
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1327, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
17:08:57 ERROR | Failed to run external process
Traceback (most recent call last):
  File "runtests.py", line 96, in fetch_mozharness
    return subprocess.check_call(command, env=self.env)
  File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
CalledProcessError: Command '['python', 'archiver_client.py', 'mozharness', '--repo', 'releases/mozilla-esr45', '--rev', 'default', '--debug']' returned non-zero exit status 1
```

**Comment by whimboo** [comment]
Ok, so this is broken because we use `default` as revision here and in this case it refers to the changeset `c8cdcfb2d88b` which doesn't have any builds.

https://treeherder.mozilla.org/#/jobs?repo=mozilla-esr45&revision=c8cdcfb2d88b

So we would have to find the last changeset with a successful build. This would cause quiet an amount of work which I don't really want to do for a branch which is close to be desupported soon. I think we should get issue #769 fixed first and then pin the mozharness revision to that changset.

**Comment by whimboo** [comment]
Issue #769 has been fixed by a check-in of the necessary patches on mozilla-release and mozilla-esr45. Given that it will not land for mozilla-esr38 I'm going to create a patch which will add a fixed revision (226acde614dc) for mozilla-esr38 on the mozilla-esr45 branch.

**Comment by whimboo** [comment]
PR #773 has been merged to master and will be pushed to both staging and production soon.
